src/unitycatalog/ai/utils.py

```diff
@@ -113,3 +141,159 @@ def validate_full_function_name(function_name: str) -> FullFunctionName:
             f"Invalid function name: {function_name}, expecting format <catalog_name>.<schema_name>.<function_name>."
         )
     return FullFunctionName(catalog_name=splits[0], schema_name=splits[1], function_name=splits[2])
+
+
+def uc_type_json_to_pydantic_type(uc_type_json: Union[str, Dict]) -> Type:
```

Comment:
```suggestion
def uc_type_json_to_pydantic_type(uc_type_json: Union[str, Dict[..., ...]]) -> Type:
```

can you add the key and value type?